src/os/ui/layer/defaultlayerui.js

```diff
@@ -433,6 +433,14 @@ os.ui.layer.DefaultLayerUICtrl.prototype.onRefreshChange = function() {
  */
 os.ui.layer.DefaultLayerUICtrl.prototype.reset = function(key) {
   var defaultValue = this.defaults[key];
+  var nodes = this.getLayerNodes();
+  for (var i = 0, n = nodes.length; i < n; i++) {
+    var value = nodes[i].getLayer().getLayerOptions()['defaults'][key];
+    if (value) {
```

Comment:
Two things here:
- The default value could be 0, so this needs to be `value != null`.
- This for loop is overwriting `defaultValue` on each iteration, which results in applying the default for the last layer in the array to all layers. It should either reset each layer individually to its own default, or use the first node in the array (we use the first to populate the control when multiple layers are selected).

@wallw-bits what are your thoughts on the multi-select behavior? I'm thinking each layer should be reset to its own default value, then the control value updated to reflect the first layer.